ansible/roles/test/tasks/common_tasks/reboot_sonic.yml

```diff
@@ -0,0 +1,23 @@
+- name: reboot
+  become: true
+  shell: shutdown -r now "Reboot SONiC."
```

Comment:
Might as well make message more descriptive, e.g.:

`shutdown -r now "Warning! System is being rebooted remotely by reboot_sonic.yml."`